src/object/srv_obj.c

```diff
@@ -2196,6 +2200,28 @@ obj_ioc_begin_lite(uint32_t rpc_map_ver, uuid_t pool_uuid,
 		D_GOTO(out, rc = -DER_STALE);
 	} else if (DAOS_FAIL_CHECK(DAOS_DTX_STALE_PM)) {
```

Comment:
minor, the FAIL_CHECK commonly is the last branch

Reply:
I will fix it.